tests/models/registry.py

```diff
@@ -193,6 +193,7 @@ def check_available_online(
     "XverseForCausalLM": _HfExamplesInfo("xverse/XVERSE-7B-Chat",
                                          is_available_online=False,
                                          trust_remote_code=True),
+    "Zamba2ForCausalLM": _HfExamplesInfo("Zyphra/Zamba2-7B-instruct"),
```

Comment:
```suggestion
    "Zamba2ForCausalLM": _HfExamplesInfo("Zyphra/Zamba2-7B-instruct",
                                         min_transformers_version="4.49"),
```

I think we can merge this PR first but without updating transformers (expecting users to install a later version). By setting `min_transformers_,version`, we can skip testing this model in CI for now.

Reply:
Sounds good